Update pictures in user-guide/access-control/ldap (#5263)
DaoCloud/DaoCloud-docs: docs/en/docs/ghippo/user-guide/access-control/ldap.md

# LDAP

> The full name of LDAP in English is Lightweight Directory Access Protocol, which is an open and neutral

Reviewer: ```suggestion
The full name of LDAP is Lightweight Directory Access Protocol, which is an open and neutral
```

> industry-standard application protocol that provides access control and maintains directory information

Reviewer: ```suggestion
industry-standard application protocol that provides access control and maintains directories
```

!!! note

    1. After you have established a trust relationship between the enterprise user management system
       and DCE 5.0 through the LDAP protocol, you can synchronize the users or groups in the enterprise
       user management system to DCE 5.0 at one time through auto/manual synchronization.
    1. After synchronization, the administrator can authorize groups/groups in batches, and users
       can log in to DCE 5.0 through the account/password in the enterprise user management system.

>     1. See the [LDAP Operations Demo Video](../../../videos/ghippo.md#ldap) for a hands-on tutorial.

Reviewer: ```suggestion
    1. See the [LDAP Operations Demo Video](../../../videos/ghippo.md#integration-with-ldap-users-and-user-groups) for a hands-on tutorial.
```
中英锚点不同
